packages/react-core/src/components/TextInput/TextInput.tsx

```diff
@@ -215,6 +215,7 @@ export class TextInputBase extends React.Component<TextInputProps, TextInputStat
           type={type}
           value={this.sanitizeInputValue(value)}
           aria-invalid={props['aria-invalid'] ? props['aria-invalid'] : validated === ValidatedOptions.error}
+          aria-expanded={isExpanded}
```

Comment:
This is more to do with the default value of the isExpanded prop, but the default value should be undefined rather than false. Right now every instance of a Textinput is going to render with the `aria-expanded` attribute due to the default `false` value, but we would only want that if the text input has some sort of expanding functionality.